app/views/validation_mailer/inform.html.erb

```diff
@@ -0,0 +1,2 @@
+<h2>Hello <%=@user%>!</h2>
+<p>Click <%= link_to "here", "/users/update", method: :post %> to validate your account</p>
```

Comment:
path might need a root_path attached. Working through how default url in environment is working